documentation/modules/rn-2.5.adoc

```diff
@@ -241,57 +252,65 @@ spec:
 
 This issue is resolved in {project-short} {project-version}, warm migration does not fail when an operation that locks the VM is performed in {rhv-short}. The migration does not fail, but starts when the VM is unlocked. link:https://issues.redhat.com/browse/MTV-687[(MTV-687)]
 
 .Deleting migrated VM does not remove PVC and PV
 
-In previous releases of {project-short}, when removing a VM that was migrated, its persistent volume claims (PVCs) and physical volumes (PV) were not deleted.
+In earlier releases of {project-short}, when removing a VM that was migrated, its persistent volume claims (PVCs) and physical volumes (PV) were not deleted.
 
 This issue is resolved in {project-short} {project-version}, PVCs and PVs are deleted when deleting migrated VM.link:https://issues.redhat.com/browse/MTV-492[(MTV-492)]
 
 .PVC deletion hangs after archiving and deleting migration plan
 
-In previous releases of {project-short}, when a migration failed, its PVCs and PVs were not deleted as expected when its migration plan was archived and deleted.
+In earlier releases of {project-short}, when a migration failed, its PVCs and PVs were not deleted as expected when its migration plan was archived and deleted.
 
 This issue is resolved in {project-short} {project-version}, PVCs are deleted when archiving and deleting migration plan.link:https://issues.redhat.com/browse/MTV-493[(MTV-493)]
 
-.VM with multiple disks may boot from non-bootable disk after migration
+.VM with multiple disks can boot from a non-bootable disk after migration
 
-In previous releases of {project-short}, VM with multiple disks that were migrated might not have been able to boot on the target {ocp} cluster.
+In earlier releases of {project-short}, VM with multiple disks that were migrated might not have been able to boot on the target {ocp} cluster.
 
 This issue is resolved in {project-short} {project-version}, VM with multiple disks that are migrated are able to boot on the target {ocp} cluster. link:https://issues.redhat.com/browse/MTV-433[(MTV-433)]
 
-
 .Transfer network not taken into account for cold migrations from vSphere
+
 In {project-short} releases 2.4.0-2.5.3, cold migrations from vSphere to the local cluster on which {project-short} was deployed did not take a specified transfer network into account. This issue is resolved in  {project-short} 2.5.4.  link:https://issues.redhat.com/browse/MTV-846[(MTV-846)]
 
-For a complete list of all resolved issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12424644[Resolved Issues] in Jira.
+.Fix migration of VMs with multi-boot guest operating system from vSphere 
+
+The virt-v2v arguments include `–root first`, which mitigates an issue with multi-boot VMs where the pod fails. Adding this argument back to the entrypoint will resolve the issue. Link:https://issues.redhat.com/browse/MTV-968[(MTV-968)]
```

Comment:
```suggestion
In {project-short} 2.5.6, the virt-v2v arguments include `–root first`, which mitigates an issue with multi-boot VMs where the pod fails. This is a fix for a regression that was introduced in {project-short} 2.4 in which the '--root' argument was dropped. Link:https://issues.redhat.com/browse/MTV-968[(MTV-968)]
```
(I don't think we should mention 'entrypoint', we can indicate that it's a fix for a regression though

Reply:
Done